README.md

```diff
@@ -108,42 +129,74 @@ could attach the time of pushing as a timestamp.)
+of `job`:
+
     /metrics/jobs/<JOBNAME>[/instances/<INSTANCENAME>]
 
-`<JOBNAME>` is used as the value of the `job` label, and `<INSTANCE>`
-as the value of the `instance` label. The instance part of the URL is
-optional. If it is missing, the IP number of the pushing host is used
-as the value for the 'instance' label instead.
+If this version of the URL path is used _with_ the `instances` part,
+it is equivalent to the URL path above with an `instance` label, i.e.
 
-If those labels are already set in the body of the request (as regular
-labels, e.g. `name{job="foo",instance="bar"} 42`), _the values of
-those labels will be overwritten with values determined as described
-above!_ (This behavior might be changed in the future if a valid
-use-case can be shown.)
+    /metrics/jobs/foo/instances/bar
 
-### `POST` method
+is equivalent to
+
+    /metrics/job/foo/instance/bar
+
+(Note the missing pluralizations.)
+
+However, if the `instances` part is missing, the Pushgateway will
+automatically use the IP number of the pushing host as the 'instance'
+label, and grouping happens by 'job' and 'instance' labels.
 
-`POST` is used to add metrics to previously pushed metrics. Note that
-only previously pushed metrics with a different metric name, job label
-or instance label are preserved. Each `POST` will completely replace
-existing metrics with the same metric name, job label, and instance
-label as the metrics pushed, even if the existing metrics had a
-different label set otherwise.
+Example: Pushing metrics from host 1.2.3.4 using the deprecated URL path
 
-The response code upon success is always 202 (even if that same metric has
-never been pushed before, i.e. there is no feedback to the client if
-the push has replaced a metric or created a new one).
+    /metrics/jobs/foo
 
-The body of the request contains the metrics to push either as delimited binary protocol
-buffers or in the simple flat text format (both in version 0.0.4, see the
-[data exposition format specification](https://docs.google.com/document/d/1ZjyKiKxZV83VI9ZKAXRGKaUKK2BIWCT7oiGBKDBpjEY/edit?usp=sharing). Discrimination between the two variants is done via content-type
-header. (In case of an unknown content-type, the text format is tried as a fall-back.)
+is equivalent to pushing using the URL path
+
+    /metrics/job/foo/instance/1.2.3.4
+
+### `PUT` method
+
+`PUT` is used to push a group of metrics. All metrics with the
+grouping key specified in the URL are replaced by the metrics pushed
+with `PUT`.
+
+The body of the request contains the metrics to push either as
+delimited binary protocol buffers or in the simple flat text format
+(both in version 0.0.4, see the
+[data exposition format specification](https://docs.google.com/document/d/1ZjyKiKxZV83VI9ZKAXRGKaUKK2BIWCT7oiGBKDBpjEY/edit?usp=sharing)).
+Discrimination between the two variants is done via the content-type
+header. (In case of an unknown content-type, the text format is tried
```

Comment:
One could argue that should be "content type"... but I reworded it to refer to the header name, too.